paf-mvp-core-js/src/log.ts

```diff
@@ -9,22 +11,37 @@ export class Log {
   }
 
   public Debug(...args: unknown[]) {
+    if (!Log.enable) {
```

Comment:
FAR: would be better to use levels to toggle different logs ON / OFF.

Reply:
I thought about it but it may resolve only partially the problem. It will certainly reduce the number of logs, but we may still have error logs in the console that are actually "normal" in the tests.

I think we should consider using a dedicated library for that configurations.

Reply:
My point was simply to toggle all levels to "inactive" for tests, which would have the same effect, but would open the possibility to have finer grain configuration for prod.

Reply:
Make sense. I buy it.

Reply:
Done.